packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -88,7 +89,8 @@ export class OpcFeedback extends LitElement {
           category: (this._error) ? 'BUG' : 'FEEDBACK',
           stackInfo: {
             "stack": navigator.appVersion,
-            "path": window.location.pathname,
+            "path": ((this._path.length !== 1) && (this._path.substr(this._path.length-1, 1) === '/') || (this._path.substr(this._path.length-1, 1) === '#')) ? this._path.slice(0, -1) : this._path
```

Comment:
can you document what this validation does?

Reply:
This actuallly truncates the last chracter `/` and `#` from end of URL.

Reply:
Ohh okay, can you add documentation?

Reply:
Included the doc